imageregistry/v1/types.go

```diff
@@ -306,6 +306,59 @@ type ImageRegistryConfigStorageIBMCOS struct {
 	ServiceInstanceCRN string `json:"serviceInstanceCRN,omitempty"`
 }
 
+type EndpointAccessibility string
+
+var (
+	InternalEndpoint = EndpointAccessibility("Internal")
+	PublicEndpoint   = EndpointAccessibility("Public")
+)
+
+type Encryption string
+
+var (
+	ClearText = Encryption("ClearText")
```

Comment:
usual style is `ClearText Encryption = "ClearText"`

Reply:
and they should be defined as `const`, not `var`.